src/components/CreateAccount.jsx

```diff
@@ -1,9 +1,287 @@
-import React from 'react'
+import React from "react";
+import { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import Grid from "@mui/material/Grid";
+import { Box, MenuItem, Typography } from "@mui/material";
+import Container from "@mui/material/Container";
+import {
+  FormControl,
+  InputLabel,
+  Input,
+  FormHelperText,
+  Button,
+  Select,
+} from "@mui/material";
+import { TextField } from "@mui/material";
+import CreateAccountImage from "../assets/create-account.jpg";
+import BrandLogo from "../assets/fiftyfive-logo.png";
+import MyButton from "../components/Button/MyButton";
+import { useForm } from "react-hook-form";
+import { createAccountPageStyles } from "../CreateAccount.styles.js";
+import { useToast } from '@chakra-ui/react'
 
 const CreateAccount = () => {
+  const { classes } = createAccountPageStyles();
+  
+  const [userAccountData, setUserAccountData] = useState({});
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const navigate = useNavigate();
+
+  const onSubmit = async (data) => {
```

Comment:
what is the purpose of adding this onSubmit?, this is only consoling

Reply:
I believe you don't have the most recent code as onSubmit is a function that creates user's account on form submit !le me push the most recent code.